src/Symfony/Component/Validator/Constraints/AbstractComparisonValidator.php

```diff
@@ -36,7 +47,16 @@ public function validate($value, Constraint $constraint)
             return;
         }
 
-        $comparedValue = $constraint->value;
+        if ($path = $constraint->propertyPath) {
+            $object = $this->context->getObject();
```

Comment:
Hmm. Indeed. What should we do in such case? I guess we can only `return;` as there is no way to get the value to compare to from `null` and should not create a violation (it's the `NotNull` constraint role). Right?